src/child/containers/toolbar/Toolbar.js

```diff
@@ -78,7 +78,7 @@ class Toolbar extends Component {
         const { windowState } = this.props;
         return (
             <div className="toolbarWrapper drag">
-                <img id="logo" src={icon} className="logo" alt="Scott Logic" />
+                <img id="logo" src={icon} className={`logo${windowState.isCompact ? ' hidden' : ''}`} alt="Scott Logic" />
```

Comment:
Similar comment to above, about hiding the chart - if this needs to be hidden, can it be done in the styles?